frontend/interfaces/invite.ts

```diff
@@ -9,6 +9,7 @@ export interface IInvite {
   email: string;
   name: string;
   sso_enabled: boolean;
+  two_factor_authentication_enabled?: boolean;
```

Comment:
I see `mfa_enabled` in @iansltx's PR
here(https://github.com/fleetdm/fleet/pull/24273/files?diff=split&w=0#diff-a391798901c9c0c2e7df83982059276d57dd9b3337a39153a0f48bf2b6249dd6R29),
which is a bit more succinct while still being informative. Can we change to `mfa_enabled` for the
types here?